hyperspy/io_plugins/fei.py

```diff
@@ -503,17 +537,21 @@ def ser_reader(filename, objects=None, verbose=False, *args, **kwds):
                             'Dim-%i_CalibrationOffset' % (i + 1)][0],
                         'scale': header[
                             'Dim-%i_CalibrationDelta' % (i + 1)][0],
-                        'units': header['Dim-%i_Units' % (i + 1)][0],
+                        'units': header['Dim-%i_Units' % (i + 1)][0] if header['Dim-%i_UnitsLength'%(i+1)]>0 else 'a.u.',
```

Comment:
Yes, the units seem to be in SI, but here 'a.u.' is for the case of image stacks (in this case there is no units and UnitLength is then 0). However, it should be an unit of time, but I don't know how to get the scale.
I don't know what to put instead of 'a.u.', any better suggestions?